ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -39,297 +35,339 @@
+  <macro name="author-bib">
+    <choose>
+      <if type="interview">
+        <names variable="interviewer">
+          <name form="long" and="text" delimiter-precedes-last="never" name-as-sort-order="first" sort-separator=" " font-style="normal">
             <name-part name="family" font-variant="small-caps"/>
           </name>
-          <label form="short" prefix="&#160;(" suffix=".)"/>
         </names>
-      </else-if>
+      </if>
+      <else>
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name form="long" and="text" delimiter-precedes-last="never" name-as-sort-order="first" sort-separator=" " font-style="normal">
+                <name-part name="family" font-variant="small-caps"/>
+              </name>
+            </names>
+          </if>
+          <else-if variable="editor">
+            <names variable="editor">
+              <name form="long" and="text" delimiter-precedes-last="never" name-as-sort-order="first" sort-separator=" " font-style="normal">
+                <name-part name="family" font-variant="small-caps"/>
+              </name>
+              <label form="short" prefix="&#160;(" suffix=".)"/>
+            </names>
+          </else-if>
+        </choose>
+      </else>
     </choose>
   </macro>
-  <macro name="author-bib">
+  <!-- MACRO EDITEUR SCIENTIFIQUE (ndbp et biblio) -->
+  <macro name="editor">
     <choose>
-      <if variable="author">
-        <names variable="author">
-          <name form="long" and="text" name-as-sort-order="first" sort-separator=" " font-style="normal">
-            <name-part name="family" font-variant="small-caps"/>
+      <if type="interview">
+        <names variable="author" prefix=" entretien avec ">
+          <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " initialize-with="." font-style="normal">
+            <name-part name="family"/>
           </name>
         </names>
       </if>
-      <else-if variable="editor">
+      <else>
         <names variable="editor">
-          <name form="long" and="text" name-as-sort-order="first" sort-separator=" " font-style="normal">
-            <name-part name="family" font-variant="small-caps"/>
+          <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " initialize-with="." font-style="normal">
+            <name-part name="family"/>
           </name>
           <label form="short" prefix="&#160;(" suffix=".)"/>
         </names>
-      </else-if>
+      </else>
     </choose>
   </macro>
-  <macro name="editor">
-    <names variable="editor">
+  <!-- MACRO AUTEUR DU LIVRE (ndbp et biblio)-->
+  <macro name="book_author">
+    <names variable="container-author">
       <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " initialize-with="." font-style="normal">
-        <name-part name="family" font-variant="small-caps"/>
+        <name-part name="family"/>
```

Comment:
if you don't need the formatting, you don't need to add a `name-part` condition at all. Just delete this line and the next (i.e. 126 and 127)  and then put a / before the > sign in the line above (125)